data/src/main/java/com/globant/equattrocchio/data/ImagesServicesImpl.java

```diff
@@ -15,24 +21,25 @@
 public class ImagesServicesImpl implements ImagesServices {
 
     private static final String URL= "http://splashbase.co/";
-    private Gson gson = new Gson();
+    SplashbaseApi api;
+    private ResultDataMapper resultDataMapper;
+    private ImageDataMapper imageDataMapper;
 
-    @Override
-    public void getLatestImages(final Observer<String> observer) {
+    public ImagesServicesImpl() {
         Retrofit retrofit = new Retrofit.Builder().
                 baseUrl(URL).
                 addConverterFactory(GsonConverterFactory.create())
                 .build();
```

Comment:
it good practics use:  addCallAdapterFactory(RxJava2CallAdapterFactory.create())